libbpf-tools/wakeuptime.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+
+#ifndef __WAKEUPTIME_H
+#define __WAKEUPTIME_H
+
+#define MAX_ENTRIES 102400
+#define TASK_COMM_LEN 16
+
+struct key_T {
```

Comment:
This naming is weird, just `key` or `key_t` ?

Reply:
fixed in 5209cddb8a68d203a0b95eb8622150e5068c0265